### PR TITLE
TeachingBubble: Updated animation

### PR DESCRIPTION
**Pull request by lynamemi**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [X] Include a change request file using `$ npm run change`

#### Description of changes

(give an overview)

Worked with Carlos and Colton to update the TeachingBubble animation.

I'm not loading a gif for now because my gif tool slows down the frame rate, and it's not pretty (or accurate). I do have an mp4 I can mail out that's much smoother.

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.scss [hunk 1]
```diff
@@ -7,11 +7,64 @@
 // --------------------------------------------------
 // Teaching Bubble styles
 
+@keyframes bounceAnimation {
+  0% { transform: matrix3d(0.5, 0, 0, 0, 0, 0.5, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1); }
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.scss]: is this a reusable animation?

Comment (lynamemi) [reply packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.scss]: in what way?

**Comment by lynamemi** [comment]
@micahgodbolt - I know we discussed offline, but can I get official codeowner review from you?

**Comment by nero120** [comment]
If people prefer the original simple display animation, is there a prop that can be used to toggle the animation?

**Comment by lynamemi** [comment]
Since the animation is in css, you should be able to override it.

**Pull request merged**